docker-compose.ci.yml

```diff
@@ -0,0 +1,9 @@
+version: "3"
```

Comment:
Can this go in the docker/directory?

Reply:
Moved it under compose/ as discussed in https://github.com/google/fuzzbench/pull/521